docs/src/main/asciidoc/getting-started-dev-services.adoc

```diff
@@ -0,0 +1,326 @@
+----
+@Entity
+public class Greeting extends PanacheEntity {
+String name;
+}
+----
+
+The entity makes use of xref:{doc-guides}hibernate-orm-panache.adoc[Panache], a layer on top of Hibernate.
+ Extending `PanacheEntity` brings in a range of methods for reading, writing, and finding data.
+ Because all the data access methods are on the `Greeting` entity, rather than on a separate data access class,
+ this is an example of the active record pattern.
+
+The `Greeting` table will have one column, a field called `name`.
+
+=== Writing data
+
+To use the new entity, update the `hello` method to start writing some data.
+
+Change the method to the following:
+
+----
+@GET
+@Transactional
+@Produces(MediaType.TEXT_PLAIN)
+public String hello(@QueryParam("name") String name) {
+   Greeting greeting = new Greeting();
+   greeting.name = name;
+   greeting.persist();
+   return "Hello " + name;
+}
+----
+
+Don't forget the `@Transactional` method, which ensures writes are wrapped
+in a transaction.
+
+Generally, you shouldn't do state updates on a `GET` REST method, but here it makes
+trying things out simpler. Let's assume what's being written is an logging "side effect",
+rather than a meaningful state changes!
+
+Try out the updated endpoint by visiting http://localhost:8080/hello?name=Bloom.
+You should see a "Hello Bloom" message.
+
+=== Reading data
+Although the new persistence code seems to be working without errors, how
+do you know anything is being written to the database?
+
+Add a second REST method to `GreetingResource`.
+
+----
+@GET
+@Path("names")
+@Produces(MediaType.TEXT_PLAIN)
+public String names() {
+List<Greeting> greetings = Greeting.listAll();
+   String names = greetings.stream().map(g-> g.name)
+   .collect(Collectors.joining (", "));
+   return "I've said hello to " + names;
+}
+----
+
+To try it out, visit http://localhost:8080/hello?name=Bloom, and then http://localhost/hello/names.
+
+You should see the following message: "I've said hello to Bloom".
+
+== Dev services
+
+Reading and writing to the database seems to be working well, but that's a bit unexpected.
+Where did a PostgresQL database come from? You didn't set anything up.
+
+The database is being managed using xref:{docfile}/dev-services.adoc[dev services].
+Dev services take care of stopping and starting services needed by your application.
+Because you
+included the `postgresql-jdbc` dependency, the database is a containerised PostGresQL database.
+If you'd added `postgresql-mysql` insead, you would have gotten a containerised MySQL database.
+
+If you like, use your container tool to see what containers are running.
+For example, if you're using Docker, run `docker ps`, and for podman, run `podman ps`.
+You'll see something like the following:
+
+----
+ff88dcedd899  docker.io/library/postgres:14  postgres -c fsync...  1 hour ago  Up 1 hour  0.0.0.0:34789->5432/tcp  nostalgic_bassi
+----
+
+Stop Quarkus and run `docker ps` again.
+You should see nothing running.
```

Comment:
ok, not quite... this is relying on the watch dog, that you may have turned off with podman. It can take a few seconds.

Reply:
Hopefully, if people followed our podman guide, they are running with rootful podman, rather than with ryuk disabled. But I take your point.